client/functions.lua

```diff
@@ -96,13 +161,7 @@ end
 function QBCore.Functions.HideBlip(id, toggle)
     local blip = Blips[id]
     if not blip then return end
-    if toggle then 
-        SetBlipAlpha(blip.blip, 0)
-        SetBlipHiddenOnLegend(blip.blip, true)
-    else
-        SetBlipAlpha(blip.blip, 255)
-        SetBlipHiddenOnLegend(blip.blip, false)
-    end
+    if toggle then SetBlipAlpha(blip, 0) else SetBlipAlpha(blip, 255) end
```

Comment:
This just sets the alpha? At 0 alpha it will still be on the legend in the escape menu so won't be fully hidden